src/pages/TestCasePage/TestCasePage.tsx

```diff
@@ -63,6 +66,11 @@ const TestCasePage = () => {
       0
     );
 
+    logEvent(analytics, 'run_test_cases', {
```

Comment:
I know it would be cool to log the exact test cases they have selected, but I don't think this will fit in 100 characters :(

Reply:
Where does it say we can only use 100 characters for our fields? I guess its a nice to have though.

Reply:
https://support.google.com/firebase/answer/9237506?hl=en
"Length of event-parameter value = 100 characters"

Reply:
We could report the index/id of the test case if not student-generated

Reply:
We don't currently have a current index/id of a test case?

Reply:
Unless you mean array index but that's not reliable

Reply:
Yeah youre right, lets see what Paul thinks on Monday. I'm happy with this as-is

Reply:
Wait we don't even have an array index bc we store test cases as a map